### ISSUE TITLE
Clear 29460 with GDM on Nnvidia GPU won't boot up...

### ISSUE THREAD

**Issue by bktan8** [clearlinux/distribution#746]
Clear 29460 with Ndvidia GPU won't boot to GDM.  I can't even bring up the TTY console.  Seems to just hang with blank screen and single cursor on top-left cornner and won't respond.    

If I go back to 29350, it works fine.  

Clear 29460 works fine with AMD GPU.

**Comment by znmeb** [comment]
You might try adding `modprobe.blacklist=nouveau` to the kernel command line when you boot. The `nouveau` open source driver in the kernel can't deal with newer NVidia cards - my HP Omen (1050Ti) needs this to boot in almost every Linux LiveCD.

**Comment by lebensterben** [comment]
@bktan8 I've similar issue and I fallback to the integrated GPU of my motherboard.
Try to reinstall the GPU.

**Comment by ahkok** [comment]
@bktan8 I assume this is the `-native` kernel?

**Comment by mdhorn** [comment]
@ahkok  yes, kernel-native. That is what we use in the live-desktop ISO image.

**Comment by miguelinux** [comment]
CL 29350 is linux	5.0.14	753
CL 29460is linux	5.0.16	756

Just a CVE and an enabled writeback throttling is the difference, well the version too.

I dont see any change in nouveau in the changelogs

https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.0.15
https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.0.16

Can you test with "nomodeset" in the kernel cmdline please.

**Comment by bktan8** [comment]
@miguelinux - both `nomodeset` and `modprobe.blacklist=nouveau` (as suggested by @znmeb)  allow me to boot to text-based console.

**Comment by miguelinux** [comment]
> @miguelinux - both `nomodeset` and `modprobe.blacklist=nouveau` (as suggested by @znmeb)  allow me to boot to text-based console.  
> 

and in the text-based, if you start gdm didn't come out?

`sudo systemctl start gdm`

**Comment by bktan8** [comment]
No dice!  
If I try booting without `nomodeset` and `modprobe.blacklist=nouveau`, I can at least see the mouse point show up but then disappears.  With `nomodeset` and `modprobe.blacklist=nouveau` and then `sudo systemctl start gdm`, the graphics doesn't even attempt to start up.  I just see a blink cursor on the top-left corner.

**Comment by phmccarty** [comment]
@bktan8 Are you able to capture any relevant errors from dmesg?
